extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-config.js

```diff
@@ -200,15 +172,17 @@ describes.realWin(
     });
 
     it('does use inline config when remote src is invalid', async () => {
-      env.sandbox.stub(Services, 'xhrFor').returns({
-        fetchJson() {
-          throw new Error();
-        },
-      });
-      const result = await createAmpStoryShoppingConfig('invalidRemoteUrl');
+      const invalidURL = 'invalidRemoteURL';
+      const result = await createAmpStoryShoppingConfig(invalidURL);
       expect(result).to.deep.eql(keyedDefaultInlineConfig);
     });
 
+    it('test invalid remote config url', async () => {
+      expectAsyncConsoleError(
+        /'amp-story-shopping-config:config error determining if remote config is valid json: bad url or bad json'​​​/
+      );
```

Comment:
I think a `createAmpStoryShoppingConfig()` needs to be added to this test in order for this error to be logged